### DISCUSSION TITLE
UI: Decorators, ValidationMessage

### DISCUSSION THREAD

**Discussion by albertmink** [eclipse-platform/eclipse.platform.ui#2082] [General]
We all know modern UIs that have Validation Messages for Input Fields,
<img width="497" alt="Screenshot 2024-07-15 at 15 16 32" src="https://github.com/user-attachments/assets/d2de39e6-b803-4a41-9eaf-c2b9358f05eb">
see, https://react.fluentui.dev/?path=/docs/components-field--default#validation-message

Is there something similar possible within Eclipse?

**Comment by iloveeclipse** [comment]
Yes

**Reply by albertmink** [reply]
I'm thrilled to hear more

**Reply by iloveeclipse** [reply]
https://help.eclipse.org/latest/index.jsp?topic=%2Forg.eclipse.platform.doc.isv%2Fguide%2Fjface_fieldassist.htm&cp%3D2_0_6_4

**Comment by albertmink** [comment]
Question got answered -> closing
